### PR TITLE
Update e107_admin/users.php 

### PR DESCRIPTION
**Pull request by yesszus**
Add lang codes and fixed bb codes support.

### CODE REVIEW

File: e107_languages/English/admin/lan_admin.php [hunk 1]
```diff
@@ -425,6 +425,13 @@
 
 define("LAN_UI_USING_DATABASE_TABLE", "Using [x] database table");
 define("LAN_UI_TOTAL_RECORDS", "Total Records: [x]");
+define("LAN_UI_1_HOUR", "1 hour");
+define("LAN_UI_3_HOURS", "3 hours");
+define("LAN_UI_6_HOURS", "6 hours");
+define("LAN_UI_12_HOURS", "12 hours");
+define("LAN_UI_24_HOURS", "24 hours");
+define("LAN_UI_48_HOURS", "48 hours");
+define("LAN_UI_3_DAYS", "3 days");
```
Comment (CaMer0n) [inline e107_languages/English/admin/lan_admin.php]: These terms are used only in admin/users.php and no where else in all of e107. Therefor they should be inside lan_users.php, not lan_admin.php

Comment (yesszus) [inline e107_languages/English/admin/lan_admin.php]: Moved.

File: e107_languages/English/admin/lan_users.php [hunk 2]
```diff
@@ -268,7 +275,10 @@
 define("USRLAN_248", "Us");
 define("USRLAN_249", "Us");
 define("USRLAN_250", "Us");
-
+define("USRLAN_251", "Leave blank for no change");
+define("USRLAN_252", "Resend account activation email to unactivated users.");
+define("USRLAN_253", "Older than");
+define("USRLAN_254", "  Reset all passwords");
```
Comment (nlstart) [inline e107_languages/English/admin/lan_users.php]: Why the additional spaces in USRLAN_254?

**Comment by yesszus** [comment]
Changed.

**Comment by CaMer0n** [comment]
Thank you!

**Pull request merged**